proposals/xxxx-nested-protocols.md

```diff
@@ -0,0 +1,150 @@
+# Allow Protocols to be Nested in Non-Generic Contexts
+
+* Proposal: [SE-NNNN](NNNN-filename.md)
+* Authors: [Karl Wagner](https://github.com/karwa)
+* Review Manager: TBD
+* Status: **Awaiting review**
+* Implementation: [apple/swift#66247](https://github.com/apple/swift/pull/66247)
+* Upcoming Feature Flag: `NestedProtocols`
```

Comment:
An "upcoming feature flag" is specifically for source breaking changes; I think you should mention the feature is gated behind `-enable-experimental-feature NestedProtocols` in the **Implementation** field above.